srcpkgs/libxml2-python/template

```diff
@@ -1,10 +1,10 @@
 # Template file for 'libxml2-python'
 pkgname=libxml2-python
-version=2.9.9
-revision=3
+version=2.9.10
+revision=1
 wrksrc="${pkgname%-python}-${version}"
 build_wrksrc=python
-build_style=python-module
+build_style=python3-module
```

Comment:
this doesn't seem right. It will leave the `-python` package empty except for the license file.

Reply:
what @Hoshpak said. maybe add a subpkg that depends on python3 (meta)